package.json

```diff
@@ -27,7 +28,7 @@
   "scripts": {
     "clean": "rimraf packages/**/.coverage packages/**/dist packages/**/types",
     "prebuild": "yarn run clean",
-    "build": "node ./tooling/index.js build && yarn && yarn ws:tools build && yarn run build:tsc",
+    "build": "yarn workspace @webex/calling run build:src && node ./tooling/index.js build && yarn && yarn ws:tools build && yarn run build:tsc",
```

Comment:
Is this change being done in this PR so that we don't have to run 'yarn workspace @webex/calling run build'  command separately for tsc compilation ?

Reply:
That's correct @Kesari3008 . Didn't want to have to run a separate command just for calling.